src/main/java/org/jboss/aesh/io/FileResource.java

```diff
@@ -104,6 +105,12 @@ public boolean delete() {
     }
 
     @Override
+    public void move(Resource target) throws IOException {
+        FileResource destination = (FileResource) target;
+        Files.move(this.file.toPath(), destination.file.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
so this is the point... if file exists do nothing. is a bit weird.
i think is better keep this and add an option on extensions like mv command:
-n, --no-clobber
do not overwrite an existing file

wdyt ?

pls take a look:
![a](https://cloud.githubusercontent.com/assets/6443576/6839042/981a5214-d341-11e4-8a81-bc58d801cb9d.png)